lib/order-sign.js

```diff
@@ -75,12 +61,25 @@ class SignHelper {
       }]
     }
 
+    const opts = { broadcast: txOpts.broadcast || false }
+
+    if (txOpts.expiration) {
+      const [, lastIrreversibleBlockNumber, , refBlockPrefix] = meta
+      Object.assign(txData, {
+        expiration: txOpts.expiration,
+        ref_block_num: lastIrreversibleBlockNumber & 0xFFFF,
+        ref_block_prefix: +refBlockPrefix
+      })
+    } else {
+      Object.assign(opts, this.getSignTxOpts(txOpts))
```

Comment:
do we need to cover both cases or btoh way of caclulating the expire time?

in both cases we want a tx to become invalid after some time, would one way suffice and therefore make the codebase leaner?